<?xml version="1.0"?>
<launch>

    <node pkg="cv_camera" type="cv_camera_node" name="center_camera">
        <param name="device_id" value="0"/>
        <param name="frame_id" value="center_camera"/>
    </node>

    <node pkg="cv_camera" type="cv_camera_node" name="left_camera">
        <param name="device_id" value="1"/>
        <param name="frame_id" value="left_camera"/>
    </node>

    <node pkg="cv_camera" type="cv_camera_node" name="right_camera">
        <param name="device_id" value="2"/>
        <param name="frame_id" value="right_camera"/>
    </node>

    <!-- <node pkg="cv_camera" type="cv_camera_node" name="top_camera">
        <param name="device_id" value="3"/>
        <param name="frame_id" value="top_camera"/>
    </node> -->


    <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"/>

</launch>